lib/Game.rb

```diff
@@ -0,0 +1,70 @@
+
+class Game
+    include UserInterface
+    def initialize(board, player1, player2)
+        @board =  board
+        @player_1 = player1
+        @player_2 = player2
+        @current_player = @player_1
+    end
+    WIN_COMBO = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
+     ]
+
+    def play
+        loop do 
+            show_board(@board)
+            ask_move(@board , @current_player)
+            break if is_game_over?
+            @current_player = @current_player == @player_1 ? @player_2 : @player_1
+        end
+    end
+
+    def is_game_over?
+        if is_victory? == true
+            show_info(1)
+            return true
+        elsif(is_draw? == true)
+            show_info(0)
```

Comment:
Likewise this invocation should not reside here as `is_game_over?` is just supposed to return a boolean and not do anything else. 

What we can do is create a separate method that will call `show_info` (with the corresponding argument) depending on whether `is_victory?` or `is_draw?`. I see that the `show_info` method calls takes in 0 if we have a draw, 1 otherwise. So that logic should be outside of `is_game_over?`